daft/io/common.py

```diff
@@ -44,6 +45,48 @@ def _get_tabular_files_scan(
     else:
         raise NotImplementedError(f"Tabular scan with config not implemented: {storage_config.config}")
 
+    ### FEATURE_FLAG: $DAFT_V2_SCANS
+    #
+    # This environment variable will make Daft use the new "v2 scans" and MicroPartitions when building Daft logical plans
+    if os.getenv("DAFT_V2_SCANS", "0") == "1":
+        assert (
+            os.getenv("DAFT_MICROPARTITIONS", "0") == "1"
```

Comment:
Let's just override this to be 1 on default if `DAFT_V2_SCANS` is set

Reply:
Yeah unfortunately we use `os.getenv("DAFT_MICROPARTITIONS", "0") == "1"` at import time to hotswap our `Table` implementation, so by the time we hit this code it might be "too late" to override it